components/camel-kafka/src/test/java/org/apache/camel/component/kafka/serde/DefaultKafkaHeaderSerializerTest.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.camel.component.kafka.serde;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertArrayEquals;
+
+@RunWith(Parameterized.class)
+public class DefaultKafkaHeaderSerializerTest {
+
+    private KafkaHeaderSerializer serializer = new DefaultKafkaHeaderSerializer();
+
+    private Object value;
+    private byte[] expectedResult;
+
+    public DefaultKafkaHeaderSerializerTest(Object value, byte[] expectedResult) {
+        this.value = value;
+        this.expectedResult = expectedResult;
+    }
+
+    @Test
+    public void serialize() {
+        byte[] result = serializer.serialize("someKey", value);
+
+        System.out.println(Arrays.toString(result));
```

Comment:
Use a log and don't print on standard out please

Reply:
oh, forgot to remove